presets/exhibition-preset/src/creators/image-service-slide-creator.tsx

```diff
@@ -18,9 +18,15 @@ export const imageServiceSlideCreator: CreatorDefinition = {
   ...imageServiceCreator,
   id: "@exhibitions/image-service-creator",
   create: createImageService,
+  icon: <div>IMAGE</div>,
   tags: ["image", "exhibition-slide"],
   label: "IIIF Image",
   summary: "IIIF Image service",
+  resourceType: "Canvas",
```

Comment:
This seems to let the items show in the list, but I'm not 100% its the right config